src/AuthManager.ts

```diff
@@ -97,6 +97,11 @@ export class AuthManager extends EventEmitter<Events>{
         this.emit('authorization-complete')
     }
 
+    async changeActiveLicense(newLicense: License){
+        await this._authenticator.changeActiveLicense(newLicense)
+        this._handleAuthChange()
```

Comment:
enten en `return` eller en `await`?
```suggestion
        await this._handleAuthChange()
```

Reply:
takk!